src/index.js

```diff
@@ -128,4 +128,4 @@ function required(param) {
   throw new Error(`${param} parameter is required by SyncanoClient`)
 }
 
-export default SyncanoClient
+module.exports = SyncanoClient
```

Comment:
This should work with `require('syncano-client)` but does it work with `import client from 'syncano-client`?